docs/csharp/tour-of-csharp/classes-and-objects.md

```diff
@@ -58,11 +58,9 @@ Each member of a class has an associated accessibility, which controls the regio
 * `internal`
 	- Access limited to the current assembly (.exe, .dll, etc.)
 * `protected internal`
-	- Access limited to the containing class or classes derived from the containing class
+	- Access limited to the containing class or classes derived from the containing type within the same assembly
```

Comment:
Both the old version and the new version are wrong.

What it should say is that `protected internal` means that access is limited to:

* the containing class
* classes derived from the containing class
* types within the same assembly

Reply:
The text above is correct. I would write this as:  "Access limited to the containing class, classes derived from the containing class, or classes within the same assembly."

Reply:
@BillWagner Updated as per your review comment. You can see the required change in latest commit.